docs/spec/manifest-v2-1.md

```diff
@@ -0,0 +1,146 @@
+# Image Manifest Version 2, Schema 1 
+
+This document outlines the format of of the V2 image manifest.  Image manifests
+describe the various constituents of a docker image.  Image manifests can be
+ serialized to JSON format with the following media types:
+
+Manifest Type  | Media Type
+------------- | -------------
+manifest  | "application/vnd.docker.distribution.manifest.v1+json"
+signed manifest  | "application/vnd.docker.distribution.manifest.v1+prettyjws"
+
+*Note that "application/json" will also be accepted for schema 1.*
+
+References: 
+ 
+ - [Proposal: JSON Registry API V2.1](https://github.com/docker/docker/issues/9015) 
+ - [Proposal: Provenance step 1 - Transform images for validation and verification](https://github.com/docker/docker/issues/8093)
+
+## *Manifest* Field Descriptions
+
+Manifest provides the base accessible fields for working with V2 image format
+ in the registry.
+
+- **`name`** *string*
+
+	name is the name of the image's repository
+
+- **`tag`** *string*
+	
+	tag is the tag of the image
+	
+- **`architecture`** *string*
+
+   architecture is the host architecture on which this image is intended to 
+   run.  This is for information purposes and not currently used by the engine
+   
+- **`fsLayers`** *array*
+   
+   fsLayers is a list of filesystem layer blob sums contained in this image.
+   
+   An fsLayer is a struct consisting of the following fields
+      - **`blobSum`** *digest.Digest*
+      
+      blobSum is the digest of the referenced filesystem image layer.  A 
+      digest can be a tarsum or sha256 hash.
+ 
+   
+- **`history`** *array*
+   
+   history is a list of unstructured historical data for v1 compatibility.
+   
+   history is a struct consisting of the following fields
+   - **`v1Compatibility`** string
+   
+      V1Compatibility is the raw V1 compatibility information.  Generally	 
```

Comment:
This will always point to a string encoded Docker Image JSON blob as defined at https://github.com/docker/docker/blob/master/image/spec/v1.md. There is no other
defined use for this field so we don't have to label it as *generally* the value.